### PR TITLE
WIP: Beshare: x64 and sys provided Muscle support

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: haiku-apps/beshare/beshare-2.32.recipe [hunk 1]
```diff
@@ -53,10 +51,10 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	ln -sf ../../sources-2/muscle muscle
 	cd source
+	export CFLAGS="-I$includeDir/muscle -std=c++11 -D_BSD_SOURCE"
+	export CXXFLAGS=$CFLAGS"
```
Comment (Begasus) [inline haiku-apps/beshare/beshare-2.32.recipe]: Missing " (	export CXXFLAGS="$CFLAGS")?

Comment (Begasus) [reply haiku-apps/beshare/beshare-2.32.recipe]: That or one to many at the end :)

Comment (waddlesplash) [reply haiku-apps/beshare/beshare-2.32.recipe]: Looks misquoted?

**Comment by extrowerk** [comment]
Fixed in https://github.com/haikuports/haikuports/commit/2f0f765ce873d89bfd0853d990edd02411ddc386

**Pull request closed without merging**